app/javascript/components/patient/household/HOH.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import { Row, Table } from 'react-bootstrap';
+import ChangeHOH from './actions/ChangeHOH';
+import EnrollHouseholdMember from './actions/EnrollHouseholdMember';
+import { formatNameAlt } from '../../../utils/Patient';
+
+class HOH extends React.Component {
```

Comment:
This isnt anything major, but `HOH` just looks wrong to me. I'd prefer to see `HoH`. I know it's small, but all caps just doesnt sit entirely right with me.

Reply:
I went back and forth on this, but we have a `ChangeHOH` component so I did model it off of that.  Maybe we do `ChangeHoH` and `HeadOfHousehold`?  Is that too much? I'll default to @holmesie for this one

Reply:
I like the spelled out version myself

Reply:
updated here: https://github.com/SaraAlert/SaraAlert/pull/844/commits/6008b33f5692e53a31236bd1cff730bf28eb4c31